packages/babel-generator/src/generators/flow.js

```diff
@@ -270,6 +270,9 @@ export function ObjectTypeAnnotation(node: Object) {
     this.space();
 
     this.printJoin(props, node, {
+      addNewlines(leading) {
```

Comment:
I guess all it really does is prevent spaces from getting generated if there is no body:

```
function foo(): {} {}
declare class A<T = string> {};
```
vs.
```
function foo(): { } {}
declare class A<T = string> { };
```

If that isn't a concern, I can remove and update tests?